docker_test.go

```diff
@@ -61,13 +58,11 @@ func TestGetImage(t *testing.T) {
 					},
 				},
 			},
-			noDockerBuild: false,
-			refresh:       false,
-			useLocalImage: true,
-			result:        testImageName + ":" + testTag + "-" + "buildtag",
+			result: testImageNameTagged + "-" + "buildtag",
 		},
 		{
 			name: "With build config and no build flag",
+			args: []string{"--li", "--ndb", "-D"},
```

Comment:
By doing this, all the tests depend on argument parsing so you have a single point of failure for all your tests, this makes debugging harder. Tests shouldn't all rely on the same point of entry, but since we have like 10 tests, I guess it doesn't really matter

Reply:
Good point, I'll restore the test as it was.